docs/built-in-models/linear.md

```diff
@@ -79,16 +81,18 @@ print(
 # Output:
 #  100 examples over 100 have an FHE inference equal to the clear inference
 ```
+## Model accuracy
+We can plot the decision boundary of the classifier and compare those results with a scikit-learn model executed in clear. You can find the complete code in the [LogisticRegression notebook](../tutorials/ml_examples.md).
```

Comment:
```suggestion
The figure below shows how to plot the decision boundary of the FHE classifier and compares those results with a scikit-learn model executed in clear. You can find the complete code in the [LogisticRegression notebook](../tutorials/ml_examples.md).
```

Reply:
The figure below compares the decision boundary of the FHE classifier and a scikit-learn model executed in clear.